### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by rutu14**
Clicking on the category card on the home page will link to product page.

### CODE REVIEW

File: css/product-page.css [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    color: var(--black);
+    padding: 0 1.6rem 0.7rem;
+}
+
+.product-btns{
+    display: none;
+    width: 21rem;
+    height: 8.6rem;   
+    align-items: center;
+    transform: translate(0, 190px);
```
Comment (DivyanshSareen) [inline css/product-page.css]: can use relative units here

File: css/product-page.css [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    background-color: var(--secondary-colour);
+    border: none; 
+    border-radius: 50%;
+    height: 16px;
+    width: 16px;
+}
+  
+input[type="range"]:focus::-moz-range-thumb{
+    outline: 2px solid var(--secondary-colour);
+    outline-offset: 0.125rem;
```
Comment (Omkar-Ghate) [inline css/product-page.css]: can use px here

**Review by DivyanshSareen — APPROVED** [review]
UI looks good, good to merge

**Review by amartya-rs — APPROVED** [review]
Looks good to me !

**Review by Omkar-Ghate — APPROVED** [review]
The UI is neat, the transitions on the filters look great. Good to merge after the mentioned change.

File: css/product-page.css (hunk 2 above)
Comment (rutu14) [reply css/product-page.css]: Sure

File: css/product-page.css (hunk 1 above)
Comment (rutu14) [reply css/product-page.css]: Sure

**Pull request merged**